dist/node/client.js

```diff
@@ -65,13 +65,13 @@ var StitchClient = function () {
       baseUrl = options.baseUrl;
     }
 
-    this.clientAppID = clientAppID;
```

Comment:
i think we should just ignore it in PRs since it always needs to be freshly built for releases anyway.